token_creator.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
This seems rather unpolished approach, is there a more `ruby` way to accomplish your goal? At least I'd like to see it in `bin/scripts` with some more error handling if it is to be bash